### PR TITLE
Technics KN5000 musical keyboard

### PR DESCRIPTION
**Pull request by felipesanches**
This is a much smaller pull request with a smaller amount of code, in the hope to facilitate code-reviews. Once this lands, I'll open subsequent PRs with additional chunks, to be reviewed as well.

The full wip patchset is still available at PR #11835, but that seems to be too big a fish to fry all at once...

### CODE REVIEW

**Comment by happppp** [comment]
I'm 50/50 on the company string being the brand "Technics", but as I mentioned on Discord the other day IMO the driver file belongs in the Matsushita folder.
https://en.wikipedia.org/wiki/Technics_(brand)

**Comment by felipesanches** [comment]
I've made the changes suggested by code-reviewers (both here and also via private chat) and then squashed everything into a single commit.

For reference, here were the fixes made after code-reviews:
* some minor tweaks (remove "Keyboard" from the device title, reorder includes)
* properly declared the checking device LEDs
* moved the kn5000 driver to the `src/mame/matsushita` directory
* simplified ioport reads for the control panel buttons

**Comment by felipesanches** [comment]
@cuavas, @rb6502 and others, last week I made a few final adjustments based on the initial feedback provided by @happppp and this PR is ready for your code-review & merge.

**Comment by rb6502** [comment]
Thanks @felipesanches!

File: src/mame/matsushita/kn5000.cpp [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+	ROM_LOAD("kn5000_waveform_rom.ic304", 0x000000, 0x400000, NO_DUMP)
+	ROM_LOAD("kn5000_waveform_rom.ic305", 0x400000, 0x400000, NO_DUMP)
+	ROM_LOAD("kn5000_waveform_rom.ic306", 0x800000, 0x400000, NO_DUMP)
+	ROM_LOAD("kn5000_waveform_rom.ic307", 0xc00000, 0x400000, CRC(20ff4629) SHA1(4b511bff6625f4655cabd96a263bf548d2ef4bf7))
+ROM_END
+
+} // anonymous namespace
+
+//   YEAR  NAME   PARENT  COMPAT  MACHINE INPUT   STATE         INIT        COMPANY      FULLNAME             FLAGS
+CONS(199?, kn5000,    0,       0, kn5000, kn5000, kn5000_state, empty_init, "Technics", "SX-KN5000", MACHINE_NOT_WORKING|MACHINE_NO_SOUND)
```
Comment (rb6502) [inline src/mame/matsushita/kn5000.cpp:845]: Looks like 1998: https://www.soundonsound.com/reviews/technics-kn5000

**Comment by felipesanches** [comment]
Thanks, @rb6502!

**Pull request merged**